File: modul_praktikum_python/contoh_code/python/cek_huruf_besar.py
"""Cek apakah karakter termasuk huruf besar A..Z."""


def main() -> None:
    s = input("Masukkan Suatu Karakter :").strip()
    a = s[0] if s else ""
    print()
    if "A" <= a <= "Z":
        print("Anda menekan huruf besar")
        print("Huruf yang anda tekan, Huruf ", a)
    else:
        print("Anda tidak menekan huruf besar")
        print("Karakter yang anda tekan, Karakter ", a)
    input("Tekan Enter untuk melanjutkan...")


if __name__ == "__main__":
    main()

File: modul_praktikum_python/contoh_code/pascal/cek_huruf_besar.pas
uses crt; 
var 
  A : char; 
begin 
  write(‘Masukkan Suatu Karakter :’); 
  A:=readkey; 
  writeln; 
  if (A>=’A’)and(A<=’Z’) then  
    begin  
     writeln(‘Anda menekan huruf besar’); 
     writeln(‘Huruf yang anda tekan, Huruf ’,A); 
    end 
  else 
    begin  
     writeln(‘Anda tidak menekan huruf besar’); 
     writeln(‘Karakter yang anda tekan, Karakter ’,A); 
    end; 
  readkey; 
end.
